app/controllers/users_controller.rb

```diff
@@ -1,17 +1,19 @@
 class UsersController < ApplicationController
-  def show; end
+  def show
+    @user = UserFacade.new(current_user) if current_user.token
```

Comment:
This only displays the UserFacade if the user has a token? So the Facade is purely for Github?